FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/demos/V2BottomSheetActivity.kt

```diff
@@ -0,0 +1,333 @@
+                modifier = Modifier.fillMaxWidth()
+            ) {
+                Text(
+                    text = "Expandable",
+                    modifier = Modifier.weight(1F),
+                    color = FluentTheme.aliasTokens.neutralForegroundColor[AliasTokens.NeutralForegroundColorTokens.Foreground1].value(
+                        themeMode = ThemeMode.Auto
+                    )
+                )
+                ToggleSwitch(checkedState = expandableState,
+                    onValueChange = { expandableState = it }
+                )
+            }
+
+            Row(
+                horizontalArrangement = Arrangement.spacedBy(30.dp),
+                verticalAlignment = Alignment.CenterVertically,
+                modifier = Modifier.fillMaxWidth()
+            ) {
+                Text(
+                    text = "Show Handle",
+                    modifier = Modifier.weight(1F),
+                    color = FluentTheme.aliasTokens.neutralForegroundColor[AliasTokens.NeutralForegroundColorTokens.Foreground1].value(
+                        themeMode = ThemeMode.Auto
+                    )
+                )
+                ToggleSwitch(checkedState = showHandleState,
+                    onValueChange = { showHandleState = it }
+                )
+            }
+
+            Row(
+                horizontalArrangement = Arrangement.spacedBy(30.dp),
+                verticalAlignment = Alignment.CenterVertically,
+                modifier = Modifier.fillMaxWidth()
+            ) {
+                Text(
+                    text = "Peek Height is restricted to half of screen size.",
+                    modifier = Modifier.weight(1F)
+                )
+            }
+
+            Row(
+                horizontalArrangement = Arrangement.spacedBy(16.dp),
+                verticalAlignment = Alignment.CenterVertically,
+                modifier = Modifier.fillMaxWidth()
+            ) {
+                Text(
+                    text = "Peek Height $peekHeightState",
+                    modifier = Modifier.weight(1F),
+                    color = FluentTheme.aliasTokens.neutralForegroundColor[AliasTokens.NeutralForegroundColorTokens.Foreground1].value(
+                        themeMode = ThemeMode.Auto
+                    )
+                )
+                com.microsoft.fluentui.tokenized.controls.Button(
+                    style = ButtonStyle.Button,
+                    size = ButtonSize.Medium,
+                    text = "+ 8 dp",
+                    enabled = !hidden,
+                    onClick = { peekHeightState += 8.dp })
+
+                com.microsoft.fluentui.tokenized.controls.Button(
+                    style = ButtonStyle.Button,
+                    size = ButtonSize.Medium,
+                    text = "- 8 dp",
+                    enabled = !hidden && (peekHeightState > 0.dp),
+                    onClick = { peekHeightState -= 8.dp })
```

Comment:
Ensure peek height going negative is handled gracefully

Reply:
negative peekheight is allowed though it would not make visible the screen content in shown state.

Reply:
Last we discussed, you said this will throw an Illegal Argument error as -ve peek height won't really show up on screen. Any reason for the change of thought? Is there a legitimate use case of -ve peek?

Reply:
invoking  IllegalArgumentException would crash the host app, which won't be a desired behavior & allowing negative value is no harm as well.  And as the value can could come from any calculation & dynamically change, hence won't restrict it to work with only positive values.